File: practias_logica_programacion/python_practicas/ejercicio6_cuadradoAsteriscos.py


def cuadrado_asteriscos(num):
    ancho = '*'*(num-2)
    ancho_espacio =' '*(num-2)
    for nu in range(1,num+1):
        if nu == 1 or nu == num:
            print('*'+ancho+'*')
        else:
            print('*'+ancho_espacio+'*')
cuadrado_asteriscos(10)

File: practias_logica_programacion/javaScript_practicas/ejercicio6_cuadradoAsteriscos.js
'use strict'

function cuadradoAsteriscos(num){

    let espacio_asterisco = '*'.repeat(num-2);
    let espacio = ' '.repeat(num-2);
    for(let i=1; i<(num+1); i++){
        if(i ==1 || i == num){
            console.log('*'+espacio_asterisco+'*');
        }
        else{
            console.log('*'+espacio+'*');
        }
       
    }
 }

cuadradoAsteriscos(5)